app/src/main/java/org/mozilla/focus/Components.kt

```diff
@@ -209,9 +209,7 @@ class Components(
         )
     }
 
-    val experimentalFeatures by lazy {
-        ExperimentalFeatures(experiments)
-    }
+    var experimentalFeatures = ExperimentalFeatures { experiments }
```

Comment:
Tried avoiding setting this as `var` and also keeping the lazy initialization.
Looked into using something like a mutable lazy but it didn't really work.
I'd say it's important to have this patch qa-tested and integrated in the codebase in preparation of the live experiment and figuring a better solution for the overall flow and configuration can come later to be used also on Fenix.